lib_collections/models.py

```diff
@@ -1482,6 +1529,30 @@ def is_web_exhibit(self):
         """
         return self.web_exhibit
 
+    @property
+    def is_online_exhibit(self):
+        """
+        Determines if the exhibit is online-only.
+
+        Returns:
+            Boolean
+        """
+        space_type = self.space_type
+        if space_type == 'Online':
+            return True
+
+    @property
+    def is_physical_exhibit(self):
+        """
+        Determines if the exhibit is a physical exhibit.
+
+        Returns:
+            Boolean
+        """
+        space_type = self.space_type
+        if space_type != 'Online':
+            return True
```

Comment:
I debated creating this one, but it comes in handy when you want to say "do this if it is not online" in a quick way in the templates. You can see this in the news story page at the bottom of a story that has a related exhibit.

Reply:
I think it's fine. It's a pretty common convention. I'd just recommend reusing the other function for less duplication of code. It's also more readable because it tells other programmers that it really _is_ just an inverse of the logic (as opposed to making them read it to figure that out).